guake/test_terminal.py

```diff
@@ -26,14 +26,9 @@
 import os
 
 # pylint: disable=wrong-import-position
-import gi
-gi.require_version('Gtk', '3.0')
-gi.require_version('Notify', '0.7')
-gi.require_version('Vte', '2.91')
-
-from gi.repository import GLib
-from gi.repository import Gtk
-from gi.repository import Vte
+from guake import gi
+
+from gi.repository import GLib, Gtk, Vte
```

Comment:
I had errors about the version, that's why I had to add the require_version before, how did you manage to get rid of them?